native/machine-coding/Movie-Info-App/index.js

```diff
@@ -0,0 +1,42 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
as defer is used while loading script, I don't think we need the DOMContentLoaded listener. Can be written directly